bindata/egress-router/000-nad.yaml

```diff
@@ -13,11 +13,27 @@ spec:
     "name": "egress-router-cni-nad",
     "ip": {
       "addresses": [
-        "{{.Addresses}}"
+        "{{.Addresses}}/24"
```

Comment:
egress-router-cni is expecting a CIDR that is why the additional /24
[macvlan.go#L304](https://github.com/msherif1234/egress-router-cni/blob/master/pkg/macvlan/macvlan.go#L304)

Reply:
@msherif1234 but previously, if you did put a CIDR into the Addresses[x].IP field, that would pass through all the way to the macvlan interface (right?). But now you're limited to a /24, is that what we expect?

(again, I don't know the code much and maybe it already needs to be a /24...)

Reply:
the CIDR define this field as the following 
```
IP string `json:"ip" protobuf:"bytes,1,opt,name=ip"`
```
and validate as follows 
```
ip:
                      anyOf:
                      - format: ipv4
                      - format: ipv6
                      description: IP is the address to configure on the router's
                        interface. Can be IPv4 or IPv6.
                      type: string
```
so passing /xx will fail the CRD validation 
changing CIRD to something like this will help
```
CIDR:
                  description: CIDR defines the total range of a cluster networks
                    address space.
                  type: string
                  pattern: ^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])/([0-9]|[12][0-9]|3[0-2])$
                hostSubnetLength:
                  description: HostSubnetLength is the number of bits of the accompanying
                    CIDR address to allocate to each node. eg, 8 would mean that each
                    node would have a /24 slice of the overlay network for its pods.
                  type: integer
                  format: int32
                  maximum: 30
                  minimum: 2
```
I can change the API no prob if APIs folks are ok ?